public/src/pages/views/MemberView.tsx

```diff
@@ -175,7 +175,7 @@ export class MemberView extends React.Component<IUserViewerProps, IUserViewerSta
 
     private async handlePromote(userRel: IUserRelation) {
         if (confirm(
-            `Warning! 
+            `Warning!
```

Comment:
Please change this question to; and remove the `Warning!`:
`Are you sure you want to promote XXX to teacher status?`

Reply:
Fixed.